hedyorg/hedy
# 📜 Correct the message for keypresses

This PR changes the message displayed to the user `Waiting for a button press` to `Waiting for a keypress`. This is needed because we do not have buttons and all content refers to keys, not buttons. Also note that keypress is one word.

Fixes #3856

**How to test**
Go to level 5 and write any if-pressed program (e.g. `if x is pressed print 'x' else print 'other'`) and run it. Check that the message in the output screen is updated.
